wazo_call_logd/generator.py

```diff
@@ -186,6 +210,46 @@ def _group_cels_by_linkedid(self, cels):
         cels = sorted(cels, key=attrgetter('linkedid'))
         return groupby(cels, key=attrgetter('linkedid'))
 
+    def _group_cels_by_shared_channels(
```

Comment:
I think this function deserves its own unit tests, what do you think?

Reply:
Agreed